core/src/main/scala/kafka/cluster/Broker.scala

```diff
@@ -133,6 +134,15 @@ case class Broker(id: Int, endPoints: Map[SecurityProtocol, EndPoint]) {
     endPoints.contains(protocolType)
   }
 
+  def getNode(protocolType: SecurityProtocol): Node = {
+    val endpoint = endPoints.get(protocolType)
```

Comment:
```scala
val endpoint = endPoints.getOrElse(protocolType, throw new BrokerEndPointNotAvalableException(...))
new Node(id, endpoint.host, endpoint.port)
```
Also, we can use string interpolation instead of `format`.